Library/Formula/mdocml.rb

```diff
@@ -0,0 +1,21 @@
+class Mdocml < Formula
+  homepage "http://mdocml.bsd.lv/"
+  url "http://mdocml.bsd.lv/snapshots/mdocml-1.13.2.tar.gz"
+  sha1 "0d3c4e72214f73ee81c02e9b8863db0bc8f85aaf"
+
+  conflicts_with 'groff', :because => 'both install `groff` binaries'
```

Comment:
Hm, if the conflict is the `groff` binary, I'd rather make this formula keg-only - the OS itself comes with a `/usr/bin/groff`.